widget_driver/lib/widget_driver.dart

```diff
@@ -6,5 +6,6 @@ export 'src/widget_driver.dart';
 export 'src/widget_driver_provider.dart';
 export 'src/drivable_widget.dart';
 export 'src/mock_driver_provider.dart';
+export 'src/utils/runtime_environment_info.dart';
```

Comment:
This is the change which is needed by the `widget_driver_test` package. 
It needs this `runtime_environment_info` type so that it can correctly implement the test-helper for widgetDrivers